adapters/Vungle/VungleAdapter/VungleAdNetworkExtras.h

```diff
@@ -0,0 +1,32 @@
+//
```

Comment:
This is a duplicate file. There is a `VungleAdNetworkExtras.h` file here: https://github.com/googleads/googleads-mobile-ios-mediation/blob/master/adapters/Vungle/Public/Headers/VungleAdNetworkExtras.h

Please revert.